packages/template-retail-react-app/app/pages/cart/partials/cart-secondary-button-group.jsx

```diff
@@ -62,6 +62,7 @@ const CartSecondaryButtonGroup = ({
     isAGift = false
 }) => {
     const variant = useItemVariant()
+    const isBonusProduct = variant?.bonusProductLineItem
```

Comment:
We need to think about overlap of standard product and bonus product. @sf-emmyzhang